sidecar/src/figwheel_sidecar/system.clj

```diff
@@ -130,7 +131,13 @@
                    channel-id msg-data callback))
   (-connection-data [this]
     (server/-connection-data (:figwheel-server @system)))
-  (-actual [this] (:figwheel-server @system)))
+  (-actual [this] (:figwheel-server @system))
+  Suspendable
+  (suspend [c] c)
+  (resume [c old-c]
```

Comment:
Thanks for the tip; I'll take another pass soon.
Just so I can understand, is the concern here if a resume is called before the component is started (or after it's stopped)?